src/component/plotly-graph/inject.js

```diff
@@ -2,6 +2,9 @@ const fs = require('fs')
 const path = require('path')
 const isUrl = require('is-url')
 const semver = require('semver')
+const os = require('os')
+const curl = require('curlrequest')
```

Comment:
Did we really need another dep for this? Can the native `http` and `https` modules or even the more common [`request`](https://github.com/request/request) module do the job?